package.json

```diff
@@ -72,7 +72,7 @@
     "rimraf": "^2.5.4",
     "tslint": "latest",
     "tslint-test-config-non-relative": "file:test/external/tslint-test-config-non-relative",
-    "typescript": "^2.3.0"
+    "typescript": "next"
```

Comment:
It won't compile in typescript pre-`next` though. Could I just tag it to the specific daily version?